MapboxDirections/MBRouteOptions.swift

```diff
@@ -293,6 +298,7 @@ open class RouteOptions: NSObject, NSSecureCoding {
             URLQueryItem(name: "overview", value: String(describing: routeShapeResolution)),
             URLQueryItem(name: "steps", value: String(includesSteps)),
             URLQueryItem(name: "continue_straight", value: String(!allowsUTurnAtWaypoint)),
+            URLQueryItem(name: "roundabout_exit", value: String(!includeExitRoundaboutManeuver)),
```

Comment:
This inverts the logic so that `includeExitRoundaboutManeuver = false` gets translated into `roundabout_exit=true`, which seems like it would include the exit roundabout maneuver. Are you sure that’s correct?